modules/star.cfmod.pl

```diff
@@ -45,21 +45,22 @@
 # --mem. Return the required memory allocation.
 if($required_mem){
     if(!$run_fn){
-        print '10G';
+        print '32000000000';
         exit;
     } else {
         # Parse the run file
         my ($starting_files, $config) = CF::Helpers::parse_runfile($run_fn);
     	# estimate memory based on genome size
-        my $minmem = 32000111000;
+        my $minmem = 32000000000;
```

Comment:
Hmm, I'm not sure - will it still work without that `0.0001GB` of memory..? :laughing: